doc/design.rst

```diff
@@ -11,6 +11,14 @@ The IPython notebook is pleasantly interactive in showing us rendering of LaTeX,
 and output of code execution, but the web interface feels heavy and
 uncomfortable when writing anything longer than a few pages.
 
+*******************************
+What's wrong with the notebook?
+*******************************
+
+Using model–view–controller (MVC) terminology, the Jupyter notebook interface
```

Comment:
More detail on what the model, view and controller are in terms of the notebook?